docs/b2b-edition/specs/storefront/storefront/company.yaml

```
@@ -783,8 +785,8 @@ paths:
                         email: example@example.com
       operationId: get-companies-validations-fronted-user-emails-email
       description: |-
-        Check if user email can be used for current company，user info and available status will be respond.(fronted)
-        <br>Equivalent Storefront GraphQL API Query: `userEmailCheck`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
+        Check if the user email is valid for current company; user information and available status are front-end responses.
```

we should reference this as `storefront` instead of `front-end`

---

Simplifying this so it doesn't reference front-end. After all, the responses should be in the definition of the response, not the description.

```suggestion
        Check if the user email is valid for current company.
```